src/routes/docs/contributing/translating/+page.md

```diff
@@ -0,0 +1,106 @@
+
+## ICU message syntax
+
+You may see some strings that have programmatic syntax.
+It’s called ICU message syntax, which is the standard syntax advanced by Unicode (and the parallel ISO 10646 standard) for efficiently processing text in any language and for maintaining text data integrity
+
+### Pluralization
+
+Plural type is used to handle plural category variations, as each language has its own set of plural categories (for example, English has “one” and “other”, while Ukrainian has “one”, “few”, “many”, and “other”).
+
+The following short tags are used to determine the plural categories:
+
+- zero
+- one (singular)
+- two (dual)
+- few (paucal)
+- many (also used for fractions if they have a separate class)
+- other (required—general plural form—also used if the language only has a single form)
+
+In the Editor, you don’t have to manually add or delete plural categories to the translations you are making. Just click `Copy Source`, and the source string will be copied to the translation field with the number of plural categories right for the current target language.
+
+![image](https://github.com/0x5bfa/Website/assets/62196528/830ef3f0-945b-4dae-8381-00805c7bd590)
+
+For complete list of languages with Language Plural Rules, visit [Unicode Org web page](https://www.unicode.org/cldr/charts/45/supplemental/language_plural_rules.html).
+
+Please do care about:
+
+- `other` tag has always to be defined in any languages.
+- If there're tags that have the same value as `other` tag, they can be omitted.
+- In case of a specific translation for a learned value, you can use "=NUMBER {TEXT}". Use only for values that are not specified using predefined tags in source language (`en-US`). This is most commonly used to define for the value `0`.
+
+For example, Japanese doesn't have plural but `other` tag has to be defined though `one` and `other` tag have the same value in Japanese.
+
+### Number
+
+The purpose of the number type is to display different number values such as percentage, currency, and decimal numbers independently from the locale conventions for those. This enables adjustment of the message output to the number formats used in different locales.
+
+The placeholder `{0, number}` only specifies the resulting number format, this is never translated, it must always be kept in the same form, but can be moved to positions that require translation.
+
+Please do care about:
+
+- The placeholder `#` indicates where the value will be inserted. This allows you to define a future position according to your translation.
+
+### Remarks
+
+If you're not sure what tags should be added, don't worry. Click `i` button and see available tags, or click 'generate skelton' to see preview.
+
+![image](https://github.com/0x5bfa/Website/assets/62196528/a66edfbc-1bc1-42d0-8843-76af330fc861)
+
+![image](https://private-user-images.githubusercontent.com/84145589/336040857-ee954a9b-ef7b-4cf5-8e5e-a6394ebabb1c.png)
```

Comment:
```suggestion
![image](https://private-user-images.githubusercontent.com/84145589/336040857-ee954a9b-ef7b-4cf5-8e5e-a6394ebabb1c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc1ODY1ODIsIm5iZiI6MTcxNzU4NjI4MiwicGF0aCI6Ii84NDE0NTU4OS8zMzYwNDA4NTctZWU5NTRhOWItZWY3Yi00Y2Y1LThlNWUtYTYzOTRlYmFiYjFjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjA1VDExMTgwMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA2ZTJkYThiYzQzM2Q4YzQwOTY5OTY1ZmM1Njc2Mjk1OWRlNzkxNmY5NThkZDU3MjU4MTQxNGI2NTAzNzNiNDYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.jUppxsC03mbEii6T4l8s7_IHB6vtvCs8nyhNLFuoCvM)
```
This URL not existed, It will also have to be inserted from assets